### PR TITLE
[wip] Test parallel pod node events

### PR DESCRIPTION
**Pull request by dcbw**

### CODE REVIEW

**Comment by squeed** [comment]
that is really weird, it seems like the test pods were restarted even though they shouldn't have been.
/test e2e-aws-ovn

**Comment by squeed** [comment]
Same set of network policy tests failed. Something is wrong.

File: go-controller/pkg/factory/factory.go [hunk 1]
```diff
@@ -56,6 +73,149 @@ func (i *informer) forEachHandler(obj interface{}, f func(h *Handler)) {
...
+	i := newBaseInformer(oType, sharedInformer, stopChan)
+	i.inf.AddEventHandler(i.newFederatedHandler())
+	return i
+}
+
+func newQueuedInformer(oType reflect.Type, sharedInformer cache.SharedIndexInformer, stopChan chan struct{}) *informer {
+	i := newBaseInformer(oType, sharedInformer, stopChan)
+	i.events = make([]chan *event, numEventQueues)
+	for j := 0; j < numEventQueues; j++ {
+		i.events[j] = make(chan *event, 1)
```
Comment (squeed) [inline go-controller/pkg/factory/factory.go]: Does it make sense for the queue to be so small?

**Comment by danwinship** [comment]
@dcbw still relevant?

**Pull request closed without merging**